zhaquirks/philips/sml001.py

```diff
@@ -0,0 +1,116 @@
+"""Quirk for Philips SML001."""
+from zigpy.profiles import zha, zll
+from zigpy.quirks import CustomCluster, CustomDevice
+import zigpy.types as t
+from zigpy.zcl.clusters.general import (
+    Basic,
+    Groups,
+    Identify,
+    LevelControl,
+    OnOff,
+    Ota,
+    PowerConfiguration,
+    Scenes,
+)
+from zigpy.zcl.clusters.lighting import Color
+from zigpy.zcl.clusters.measurement import (
+    IlluminanceMeasurement,
+    OccupancySensing,
+    TemperatureMeasurement,
+)
+
+from . import PHILIPS
+from ..const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+
+class OccupancyCluster(CustomCluster, OccupancySensing):
+    """philips occupancy cluster."""
+
+    manufacturer_attributes = {
+        0x0030: ("sensitivity", t.uint8_t),
+        0x0031: ("sensitivity_max", t.uint8_t),
+    }
+
+
```

Comment:
if both devices share the same "Occupancy cluster" can you please move it to `zhaquirks/philips/__init__.py`
and then in `sml001.py` and `sml002.py` just do `from . import PHILIPLS, OccupancyCluster` ?